src/Google.Pubsub.V1/PublisherClient.cs

```diff
@@ -631,10 +638,16 @@ public virtual Publisher.IPublisherClient GrpcClient
         /// Lists matching topics.
         /// </summary>
         /// <param name="project">The name of the cloud project that topics belong to.</param>
+        /// <param name="pageToken">The token returned from the previous request.
+        /// <c>Null</c> or an empty string retrieves the first page.</param>
+        /// <param name="pageSize">The size of page to request, the response will not be larger
+        /// than this, but may be smaller. <c>Null</c> or 0 uses a server-defined page size.</param>
         /// <param name="callSettings">If not null, applies overrides to this RPC call.</param>
-        /// <returns>The RPC response.</returns>
-        public virtual IEnumerable<Topic> ListTopics(
+        /// <returns>An pageable enumerable of Topic items.</returns>
```

Comment:
Nit: An pageable?

More than nit: it's really a sequence of pages of topic items.